modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js

```diff
@@ -603,7 +605,9 @@ AUI.add(
 
 				if (condition && instance._isBinaryCondition(index) && visible) {
```

Comment:
Hey @alinedoleron just for a better visualization, is a forms pattern group the if declarations with parentheses, so can you group these declarations? ex: `if((condition) && (instance._isBinaryCondition(index)) && (visible))`